3party/jansson/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ project(jansson)
 
 include_directories(src ../../)
 
-append(CMAKE_CXX_FLAGS "-std=c++11")
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
No space between set and ```(```.

Reply:
It should be derived from the root cmake list, no?

Reply:
In original project this flag appending duplicates here: 3party/jansson, 3party/minizip, map/. That's why i've decided to leave the same structure. I thought there might be a need to compile this subprojects separate or smth. If no, i suggest put this line in top of the main CMakeLists.txt:

```
project(omim C CXX)

set (CMAKE_CXX_STANDARD 11)
```

Reply:
From my point of view there's no need to build 3party separately.